src/main/java/com/codegym/jrugotom5/service/AdvertService.java

```diff
@@ -30,4 +30,8 @@ public List<AdvertDTO> getAdvertsByDateRange(LocalDate from, LocalDate to) {
                 .map(advert -> modelMapper.map(advert, AdvertDTO.class))
                 .collect(Collectors.toList());
     }
+    public AdvertDTO update(AdvertDTO advertDTO){
+        Advert advert = modelMapper.map(advertDTO, Advert.class);
+        return modelMapper.map(advertRepository.save(advert), AdvertDTO.class);
```

Comment:
and here need to do the same, but backwards

`modelMapper.map(advertRepository.save(advert), AdvertDTO.class).setUserId( *id you got earlier* );
`